### DISCUSSION TITLE
Adding files to repository

### DISCUSSION THREAD

**Discussion by fhProgrammer777** [community/community#49183] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hello Guys,
I'm new Here.😊
I want to share my finish project on "github".
I made a new repository and I tried to add some files that relate to my project , but files don't add and when I click on "commit" button ,  I got this error :
"If the problem continues, contact the site owner.
HTTP ERROR 400"
 Am I right way for sharing my project?
What's wrong?
Can someone help me?
Github is somehow complicated, isn't that ?

**Comment by samuel-ma** [comment, marked as answer]
@fhProgrammer777 
Your right, it's kinda hard making comits on github 
It's alot easier to use the command line though
Let me know if your comfortable with using the command line
If not, I could also guide u on how to use the github CLI

**Reply by fhProgrammer777** [reply]
Hello Dear samuel,
Thanks very much for answering!
I really need help for sharing my projects on github to provide a strong resume.
I don't know anything about command line and github guidance don't help me because it was somehow unclear for me.
I would be very grateful if you could help me in sharing my completed project on github.

**Comment by szymon676** [comment]
did you specify commit name?

**Comment by samuel-ma** [comment]
@fhProgrammer777 
**### Here's a 3 step simple process**

**### STEP 1: Click on " Uploading a new file "** 
![step 1](https://user-images.githubusercontent.com/82509653/222979299-db666671-3655-48a4-9b2a-846d27cece01.png)

**### STEP 2: Click on " Choose your file ", a modal will pop up, select your files from your local computer and upload them to github**
![step2](https://user-images.githubusercontent.com/82509653/222979343-9f5a472a-1cee-459e-91ae-5bd5fe17bc18.png)

**### STEP 3: Click the commit button** 
![step3](https://user-images.githubusercontent.com/82509653/222979414-d16b4fd2-bc3d-4caa-901e-d6e4bcf6c6bd.png)

This is what your repository should look like now once everything has been done correctly
![final](https://user-images.githubusercontent.com/82509653/222979434-14d2ca8b-5f5d-48b5-8cd9-48737aed4d72.png)

**### Let me know if you've successfully uploaded your files to your repository**

**Comment by fhProgrammer777** [comment]
Hello Dear Samuel-ma , 
Thank you again for your kindness. 
I also follow these steps and I get this error:
![image](https://user-images.githubusercontent.com/30872684/223197822-9fec9a24-e752-421a-8c82-f6e37f8439eb.png)



 and when I click on the commit button I got this error: 
![image](https://user-images.githubusercontent.com/30872684/223195211-128b2c91-c59f-46fe-be40-a149ca4b29d1.png)

Because this project is not open source, I cannot share whole of the code. 
I want to upload a PDF file that has a description of the project and some video files of the project implementation. Is there a limit on the type of upload file?

**Comment by samuel-ma** [comment]
@fhProgrammer777 
What's the size of the file your trying to upload?
Cause there's a limit of 50MB that github limits you to
[Read here](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github) to learn more

If that's the case, let me know
I can give u some work arounds

**Reply by fhProgrammer777** [reply]
Hello Dear Samuel,
I am going through difficult days and your support and guidance are my encouragement.
The desired files, which include PDF, video, and code, are less than 50 megabytes, but still the problem was not solved. Is it possible that it is related to the internet and its speed? Or do I need to enter something in the commit section, which includes two boxes?

**Comment by samuel-ma** [comment]
@fhProgrammer777 
Hope everything's going well your ends
And to answer your question, am pretty sure your file size is the problem here, otherwise....everything else looks to be just fine
And yes, it's mandatory to add a commit message along with your upload

**Comment by fhProgrammer777** [comment]
Hello again , 
I search on net , 
See this link : https://stackoverflow.com/questions/40963665/cannot-upload-files-for-release-in-github
After reading this article, I became like this: 😱😱😱
Instead of users doing different solutions from changing the browser, turning off the firewall or antivirus to clearing the cache of the browsers, Github should solve this problem quickly and seriously.
Since what I did didn't work, I want to try command line, can you help me with this? I thank you again and I ask God to help you instead of the favor you did to me. Amen
